src/main/kotlin/calculator/ExpressionParser.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+import calculator.domain.Expression
+import calculator.domain.Operand
+
+class ExpressionParser {
+
+    private fun findCustomDelimiter(input: String): MatchResult? {
+        return CUSTOM_DELIMITER.toRegex().find(input)
+    }
+
+    private fun toExpression(tokens: List<String>): Expression {
```

Comment:
코틀린 감수성을 추가하면 지금 작성하신 메서드는 아래와 같이 표현할 수 있습니다~!
```suggestion
     fun List<String>.toExpression(): Expression {
        return Expression(this.map(::Operand))
    }
```